x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/trial_license_complete_tier/execution_logic/threat_match_alert_suppression.ts

```diff
@@ -1126,6 +1126,132 @@ export default ({ getService }: FtrProviderContext) => {
+
+          await indexGeneratedSourceDocuments({
+            docsCount: 60000,
+            interval: [firstTimestamp, '2020-10-28T05:35:50.000Z'],
+            seed: (index, _, timestamp) => ({
+              id,
+              '@timestamp': timestamp,
+              host: {
+                name: `host-${index}`,
+              },
+              agent: { name: 'agent-a' },
+            }),
+          });
+
+          await indexGeneratedSourceDocuments({
+            docsCount: 60000,
+            interval: [secondTimestamp, '2020-10-28T06:20:50.000Z'],
+            seed: (index, _, timestamp) => ({
+              id,
+              '@timestamp': timestamp,
+              host: {
+                name: `host-${index}`,
+              },
+              agent: { name: 'agent-a' },
+            }),
+          });
+
+          await addThreatDocuments({
+            id,
+            timestamp: firstTimestamp,
+            fields: {
+              host: {
+                name: 'host-80',
+              },
+            },
+            count: 1,
+          });
+
+          await addThreatDocuments({
+            id,
+            timestamp: firstTimestamp,
+            fields: {
+              host: {
+                name: 'host-14000',
+              },
+            },
+            count: 1,
+          });
+
+          await addThreatDocuments({
+            id,
+            timestamp: firstTimestamp,
+            fields: {
+              host: {
+                name: 'host-36000',
+              },
+            },
+            count: 1,
+          });
+
+          await addThreatDocuments({
+            id,
+            timestamp: firstTimestamp,
+            fields: {
+              host: {
+                name: 'host-5700',
+              },
+            },
+            count: 1,
+          });
+
+          const rule: ThreatMatchRuleCreateProps = {
+            ...indicatorMatchRule(id),
+            alert_suppression: {
+              group_by: ['agent.name'],
+              missing_fields_strategy: 'suppress',
+              duration: {
+                value: 300,
+                unit: 'm',
+              },
+            },
+            from: 'now-35m',
+            interval: '30m',
+          };
+
+          const { previewId } = await previewRule({
+            supertest,
+            rule,
+            timeframeEnd: new Date('2020-10-28T06:30:00.000Z'),
+            invocationCount: 2,
+          });
+
+          const previewAlerts = await getPreviewAlerts({
+            es,
+            previewId,
+            sort: ['agent.name', ALERT_ORIGINAL_TIME],
+          });
+          expect(previewAlerts.length).toEqual(1);
+          expect(previewAlerts[0]._source).toEqual({
+            ...previewAlerts[0]._source,
+            [ALERT_SUPPRESSION_TERMS]: [
+              {
+                field: 'agent.name',
+                value: ['agent-a'],
+              },
+            ],
+            // should be 7 suppressed alerts
```

Comment:
I don't think this comment is necessary as it doesn't provide any more information than the next line. What _would_ be useful is _why_ 7 alerts are expected, though.